complete/src/main/java/price/checker/domain/StockPriceId.java

```diff
@@ -0,0 +1,32 @@
+package price.checker.domain;
+
+
+import java.io.Serializable;
+import java.util.Date;
+
+
+/**
+ * This is the class containing the two fields that makes up the
+ * composite key of the StockPrice table that is stored in mySQL
+ * in a database called daily_stock_prices
+ */
+public class StockPriceId implements Serializable {

```

Comment:
Try adding lombok here as well to remove the getter, setter, constructor.  Use @Value and @Builder 
https://projectlombok.org/features/Value

Reply:
Simply adding @Data worked for the StockPriceId class -- let me know if that's okay or if @builder and @value annotations are required/what you want used for this specific class